benchmark/single-source/StringSwitch.swift

```diff
@@ -178,7 +178,7 @@ func getIndex(_ s: String) -> Int {
   case "integrate": return 153
   case "well": return 154
   case "within": return 155
-  case "developerss": return 156
+  case "developers": return 156
```

Comment:
Drat, sorry I didn't notice this one when I commented on the other case but this is also modifying benchmarking content and should be reverted. Sorry!